src/xbot/common/math/PIDManager.java

```diff
@@ -19,6 +19,7 @@
     private DoubleProperty maxOutput;
     private DoubleProperty minOutput;
     private BooleanProperty isEnabled;
+    private boolean isIMasked = false;
```

Comment:
I agree this is kind of tricky to grok on face value, I like suppressed a bit more but don't feel strongly about it.